src/components/ConversationActionItem.js

```diff
@@ -81,13 +86,13 @@ const ConversationActionItem = ({
           </Text>
         </View>
         <View style={styles.sectionActionView}>
-          {itemType === 'assignee' && thumbnail !== '' && (
+          {shouldShowUserAvatar && (
             <UserAvatar thumbnail={thumbnail} userName={name} size={18} fontSize={8} />
```

Comment:
The user avatar is not loading.

Reply:
@muhsin-k Loading for me
<img width="419" alt="image" src="https://user-images.githubusercontent.com/64252451/230613846-9fc17b46-050d-473d-bd1a-ec711d27fe6a.png">